urwid/widget.py

```diff
@@ -1412,7 +1414,7 @@ def _normalize_to_caption(self, text):
             return text
         if tu:
             return text.encode('ascii') # follow python2's implicit conversion
-        return text.decode('ascii')
+        return text.decode(sys.stdin.encoding)
```

Comment:
We'll have to ask the screen for the encoding somehow, because sys.stdin isn't always the place that we're getting input. Maybe the suggestion of only sending unicode input from the screen is a better fix.

Reply:
faced this issue on Debian 9 (python-urwid 1.3.1-2+b1), how should we move forward on this issue?